openmetadata-ui/src/main/resources/ui/src/interface/search.interface.ts

```diff
@@ -189,6 +189,20 @@ export interface SearchResponse<
 
 export type Aggregations = Record<string, { buckets: Bucket[] }>;
 
+/**
+ * Because we are using an older version of typescript-eslint, defining
```

Comment:
I will take note on my end to make this as priority next. Thanks for highlighting